tatus

```diff
@@ -0,0 +1,108 @@
+[1mdiff --git a/bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/discovery/HaywardDiscoveryService.java b/bundles/org.openhab.binding.haywardomnilogic/src/main/java/org/openhab/binding/haywardomnilogic/internal/discovery/HaywardDiscoveryService.java[m
```

Comment:
Can you remove this?

Reply:
Sorry, but this is a bit confusing.  file=tatus?   Can you elaborate?

Reply:
Not sure what this is.  Github glitch?

Reply:
I think you didn't add this file by intention. So, simply remove it.